docs/web-configurator/menu-pages/02-pin-mapping.mdx

```diff
@@ -53,7 +53,7 @@ If you do not know what pins are mapped to which button on your device, there is
 
 You can configure profiles that will change the GPIO to GP2040-CE button mappings based on what profile number you have set. This means that you can have button layouts for different use cases and change between them without the need to enter the Web Configurator and remap GPIO pins.
 
-The profile number either using the Web Configurator on the [Settings page](../../web-configurator/menu-pages/01-settings.mdx) or using a [hotkey shortcut](../../hotkeys.mdx#load-profile-1-4). There is not a default input combination associated with these `Load Profile #1-4` hotkey shortcuts so you will likely need to set them up in the Settings page under [Hotkey Settings](../../web-configurator/menu-pages/01-settings.mdx#hotkey-settings).
+The profile number can be changed by either using the Web Configurator on the [Settings page](../../web-configurator/menu-pages/01-settings.mdx) or using a [hotkey shortcut](../../hotkeys.mdx#load-profile-1-4). There is not a default input combination associated with these `Load Profile #1-4` or `Next Profile` hotkey shortcuts so you will likely need to set them up in the Settings page under [Hotkey Settings](../../web-configurator/menu-pages/01-settings.mdx#hotkey-settings).
```

Comment:
Profiles always default to #1 on first boot but will remember the last set profile between unplugs.  

There is no setting in web-config currently to set the profile to boot to.

Reply:
Good point. I added the information. I also noticed some some sentence structure that could be improved, so I took the liberty to fix that while at it. Let me know if there is anything else needs to be changed. Thanks.